docs/src/examples/solvers/DC-DC converter.jl

```diff
@@ -108,6 +108,7 @@ optimizer = MOI.instantiate(AB.UniformGridAbstraction.Optimizer)
 MOI.set(optimizer, MOI.RawOptimizerAttribute("concrete_problem"), concrete_problem)
 MOI.set(optimizer, MOI.RawOptimizerAttribute("state_grid"), state_grid)
 MOI.set(optimizer, MOI.RawOptimizerAttribute("input_grid"), input_grid)
+MOI.set(optimizer, MOI.RawOptimizerAttribute("jacobian_bound"), DCDC.jacobian_bound())
```

Comment:
How was it working before if it's missing this ?

Reply:
Hahaha! 
I asked myself the same question!
Maybe a bug somewhere. I plan to explore it soon.